mapper/tedge_mapper/src/mapper.rs

```diff
@@ -78,18 +79,36 @@ impl Mapper {
         }
         Ok(())
     }
+}
 
-    pub async fn run(self) -> Result<(), mqtt_client::Error> {
-        let errors_handle = self.subscribe_errors();
-        let messages_handle = self.subscribe_messages();
-        messages_handle.await?;
-        errors_handle
-            .await
-            .map_err(|_| mqtt_client::Error::JoinError)?;
+// This should be used for c8y as well as Azure
+pub(crate) fn is_smaller_than_size_threshold(
+    input: &[u8],
+    threshold: usize,
+) -> Result<(), MapperError> {
+    let size = std::mem::size_of_val(input);
```

Comment:
uhhhh, never seen `size_of_val` :D. Can't you just use `input.len()` ?

Reply:
Yes!